airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/ArraySection.tsx

```diff
@@ -64,13 +65,12 @@ export const ArraySection: React.FC<ArraySectionProps> = ({ formField, path, dis
     };
   }, [items, formField.properties]);
 
+  const label = (
```

Comment:
Nit: I'd either move this down into `title={<Property...` or call this `groupTitle` or something.